evol/step.py

```diff
@@ -88,8 +88,13 @@ def apply(self, population) -> Population:
             return population.log(**self.kwargs)
         return population
 
-class RepeatStep(EvolutionStep):
 
+class RepeatStep(EvolutionStep):
     def apply(self, population) -> Population:
         return population.evolve(**self.kwargs)
-    # TODO: make a nice __repr__ that is recusive for repeat step and shows repeats 'n'
\ No newline at end of file
+
+
+class CallbackStep(EvolutionStep):
```

Comment:
Would it be a good idea to have a `every=1` argument here, so that you can decide to only perform the callback once in a while?

Reply:
that makes sense. ill see how exactly to implement this.

Reply:
kind of copied what our old logger did. this should be fixed now.